platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/EventUtils.java

```diff
@@ -16,61 +16,21 @@
 
 package com.swirlds.platform.event;
 
-import com.swirlds.common.platform.NodeId;
 import com.swirlds.logging.legacy.LogMarker;
-import com.swirlds.platform.EventStrings;
 import com.swirlds.platform.internal.EventImpl;
-import com.swirlds.platform.system.events.BaseEvent;
-import com.swirlds.platform.system.events.EventConstants;
 import com.swirlds.platform.system.events.PlatformEvent;
 import edu.umd.cs.findbugs.annotations.NonNull;
-import edu.umd.cs.findbugs.annotations.Nullable;
 import java.time.Instant;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.ListIterator;
-import java.util.Objects;
-import java.util.stream.Collectors;
-import java.util.stream.StreamSupport;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 public abstract class EventUtils {
     private static final Logger logger = LogManager.getLogger(EventUtils.class);
 
-    /**
-     * Converts the event to a short string. Should be replaced by {@link EventStrings#toShortString(EventImpl)}
-     *
-     * @param event the event to convert
-     * @return a short string
-     */
-    public static String toShortString(final EventImpl event) {
-        return EventStrings.toShortString(event);
-    }
-
-    /**
-     * Convert an array of events to a single string, using toShortString() on each, and separating with commas.
-     *
-     * @param events array of events to convert
-     * @return a single string with a comma separated list of all of the event strings
-     */
-    public static String toShortStrings(final EventImpl[] events) {
-        if (events == null) {
-            return "null";
-        }
-        return Arrays.stream(events).map(EventUtils::toShortString).collect(Collectors.joining(","));
-    }
-
-    public static String toShortStrings(final Iterable<EventImpl> events) {
-        if (events == null) {
-            return "null";
-        }
-        return StreamSupport.stream(events.spliterator(), false)
-                .map(EventUtils::toShortString)
-                .collect(Collectors.joining(","));
-    }
-
     public static int generationComparator(final PlatformEvent e1, final PlatformEvent e2) {
```

Comment:
lots of changes to this file. I wonder if it's worth bringing it up to compliance with our annotations and _final_ rules?